cinema_service_db_data.json

```diff
@@ -0,0 +1,550 @@
+    "pk": 3,
+    "fields": {
+      "created_at": "2022-08-09T09:06:18.876Z",
+      "user": 1
+    }
+  },
+  {
+    "model": "cinema.order",
+    "pk": 4,
+    "fields": {
+      "created_at": "2022-08-09T09:06:18.876Z",
+      "user": 3
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 1,
+    "fields": {
+      "movie_session": 1,
+      "order": 1,
+      "row": 1,
+      "seat": 1
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 2,
+    "fields": {
+      "movie_session": 1,
+      "order": 1,
+      "row": 2,
+      "seat": 4
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 3,
+    "fields": {
+      "movie_session": 2,
+      "order": 1,
+      "row": 3,
+      "seat": 3
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 4,
+    "fields": {
+      "movie_session": 2,
+      "order": 1,
+      "row": 4,
+      "seat": 3
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 5,
+    "fields": {
+      "movie_session": 3,
+      "order": 1,
+      "row": 5,
+      "seat": 1
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 6,
+    "fields": {
+      "movie_session": 4,
+      "order": 1,
+      "row": 6,
+      "seat": 2
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 7,
+    "fields": {
+      "movie_session": 5,
+      "order": 1,
+      "row": 7,
+      "seat": 3
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 8,
+    "fields": {
+      "movie_session": 5,
+      "order": 2,
+      "row": 8,
+      "seat": 4
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 9,
+    "fields": {
+      "movie_session": 5,
+      "order": 2,
+      "row": 1,
+      "seat": 5
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 10,
+    "fields": {
+      "movie_session": 6,
+      "order": 2,
+      "row": 1,
+      "seat": 6
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 11,
+    "fields": {
+      "movie_session": 7,
+      "order": 3,
+      "row": 4,
+      "seat": 7
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 12,
+    "fields": {
+      "movie_session": 8,
+      "order": 3,
+      "row": 3,
+      "seat": 7
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 13,
+    "fields": {
+      "movie_session": 8,
+      "order": 3,
+      "row": 5,
+      "seat": 4
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 14,
+    "fields": {
+      "movie_session": 8,
+      "order": 4,
+      "row": 4,
+      "seat": 4
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 15,
+    "fields": {
+      "movie_session": 1,
+      "order": 4,
+      "row": 2,
+      "seat": 3
+    }
+  },
+  {
+    "model": "cinema.ticket",
+    "pk": 16,
+    "fields": {
+      "movie_session": 2,
+      "order": 4,
+      "row": 5,
+      "seat": 3
+    }
+  },
+  {
+    "model": "user.user",
+    "pk": 1,
+    "fields": {
+      "password": "pbkdf2_sha256$320000$oZqdjfUqKC500V7kk3kMmb$TBVlOMhfJxWrxbUnNpWZK0+rYM242g7do/YM5tkhn+M=",
```

Comment:
Ensure that the password hashes are securely generated and stored. Avoid using hardcoded passwords in production data.